Objects/longobject.c

```diff
@@ -4637,8 +4637,8 @@ long_bitwise(PyLongObject *a,
         size_z = negb ? size_b : size_a;
         break;
     default:
-        PyErr_BadArgument();
-        return NULL;
+        /* Unreachable code */
```

Comment:
I concur with @terryjreedy: the comment (here and above) is redudant, please remove it.